specs/checkbox.md

```diff
@@ -0,0 +1,128 @@
+# FAST-Checkbox
+
+## Overview
+An implementation of a checkbox as a form-connected web-component.
+
+### Use Cases
+Used anywhere an author might otherwise use an `input[type="checkbox"]`
+  
+### Features
+- form association
+- focus delegation
+
+### Risks and Challenges
+We want general feature-parity between this component and an `input[type="checkbox"]`. Most of these concerns are addressed by https://github.com/microsoft/fast-dna/pull/2723, but some stateful nuance will need to be handled by this component.
+
+### Prior Art/Examples
+- [FAST-DNA (React)](https://explore.fast.design/components/checkbox)
+- [Material UI](https://material-ui.com/components/checkboxes/)
+- [Lightning Design](https://www.lightningdesignsystem.com/components/checkbox/)
+- [Carbon Design](https://www.carbondesignsystem.com/components/checkbox/code)
+- [Ant Design](https://ant.design/components/checkbox/)
+- [Atlassian](https://atlaskit.atlassian.com/packages/core/checkbox)
+- [Office Fabric](https://developer.microsoft.com/en-us/fabric#/controls/web/checkbox)
+- [Windows (UWP)](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/checkbox)
+---
+
+### API
+Extends [form associated custom element](./form-associated-custom-element.md).
+
+*Component Name*
+`fast-checkbox`
+
+*IDL attributes*
+- `defaultChecked: boolean`
+  - If the checkbox is checked by default. Synchronized with the `checked` content attribute
+- `checked: boolean`
+  - The current checked state of the checkbox
+- `indeterminate: boolean`
+  - The indeterminate state. Independent of checked
+
+*Content attributes*
+- `readonly`
+  - The checkbox should be submitted with the form but should not be editable.
+- `value`
+  - Defaults to "on" to match `input[type="checkbox"]`
```

Comment:
"on"? I thought it was true/false?

Reply:
nope - the `checked` property is true/false. Value is "on"